biomage/rds/start_port_forwarding.py

```diff
@@ -0,0 +1,184 @@
+import click
+
+from ..utils.constants import STAGING
+
+from subprocess import PIPE, run
+
+@click.command()
+
+@click.option(
+    "-i",
+    "--input_env",
+    required=False,
+    default=STAGING,
+    show_default=True,
+    help="Input environment of the RDS server.",
+)
+
+# Disabled, only 5432 works for now
+# @click.option(
+#     "-p",
+#     "--local_port",
+#     required=False,
+#     default=5432,
+#     show_default=True,
+#     help="Local port from which to connect.",
+# )
+
+def start_port_forwarding(input_env, local_port = 5432):
```

Comment:
considering we will be using this often you could name it something shorter like `connect`? I also tried this and didn't work with the function name, I had to replace underscores for dashes which is a bit confusing `biomage rds start-port-forwarding` when looking at the code

Reply:
changed to `tunnel`